### PR TITLE
Fix invalid json in configmap

### PR DESCRIPTION
**Pull request by cliveseldon**
Typo: missing comma from recent merge.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/231)

### CODE REVIEW

File: config/default/configmap/kfservice.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ data:
         },
         "pytorch": {
             "image": "gcr.io/kfserving/pytorchserver"
-        }
+        },
```
Comment (hougangliu) [inline config/default/configmap/kfservice.yaml]: sorry to introduce this issue, please also update https://github.com/kubeflow/kfserving/blob/master/install/v0.1.0/kfserving.yaml#L516

**Comment by ellis-bigelow** [comment]
+1 for end to end integration tests.

File: config/default/configmap/kfservice.yaml (hunk 1 above)
Comment (ellis-bigelow) [reply config/default/configmap/kfservice.yaml]: Wow nice catch Hougang! 
@cliveseldon, happy to merge this in as soon as you get both of these.

**Pull request merged**